### DISCUSSION TITLE
chia shows 0 NFT plots

### DISCUSSION THREAD

**Discussion by Araquiel** [Chia-Network/chia-blockchain#8215] [Support]
so i have about 450 NFT plot for spacepool and its work fine
but yesterday i see no point on pool, restart PC and see no one plot on pool. Chia just do not show anyone plot.
see it in the end of logfile:
`failed to open file ... Invalid plot header magic Traceback (for all plots)`
and one more at the end
`ERROR Coud not find authentifikation sk for pk`

**Comment by loppefaaret** [comment]
It sounds like you got a potential hardware issue ? could you perhaps try and see if you can read the data on the drives outside of chia ? like tryingto copy a plot from from one of the drives, perhaps even one of the plotfiles reported as having a nad header magic ? perhaps try and du a plots check onto the file as well ?

**Reply by Araquiel** [reply]
how to manage farmer pk list?

**Comment by Araquiel** [comment]
@loppefaaret 
i am sucsessfully copied 2 plots to another drive.
i think trouble in a config.
reinstall chia and import old config file.
what i can see in log file:
```
harvester chia.plotting.plot_tools: WARNING  Plot F:\spacepool\plot-k32-2021-08-21-03-03-7dd76b9a44a9bb8eacee4f3ace499ba32c3d4b4e9a66225082ac360702ecd47e.plot has a farmer public key that is not in the farmer's pk list.
 farmer chia.farmer.farmer         : ERROR    Could not find authentication sk for pk: 
```
and no errors and no plots on gui.
 i think it's a key trouble but i don't know how do fix it.
 user chia 1.2.3/win 10/

**Reply by Araquiel** [reply]
run "chia plots check"
farmer PK is correct, but pool public key is empty. plotink for spacepool with key for NFT plots. its normal result if check pool NFR plots?

**Reply by loppefaaret** [reply]
pool public key is `none`on any plot that belongs to a plotNFT
have you at any point removed keys, or plotted on other machines, that had other keys on them ?

**Reply by Araquiel** [reply]
problem is solved:
reinstall do not effect.
reinstall with deleting keys on main screen and clean all chia files with full resync of chain - problem solved. after sync is complete my plots works fine again.
i think problem was in a key configure. but i do not know where its store on the PC.
i think delete keys and reimport seed phrase without backuping can solve its problem too.

**Comment by BrandtH22** [comment]
@Araquiel , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
